conans/test/functional/build_helpers/meson_test.py

```diff
@@ -10,6 +10,7 @@ class MesonTest(unittest.TestCase):
 
     @pytest.mark.skipif(platform.system() != "Windows", reason="Needs windows for vcvars")
     @pytest.mark.visual_studio
+    @pytest.mark.tool_meson
```

Comment:
```suggestion
```

Reply:
It is good to move it to ``functional``, as it requires Visual Studio installed, for the vcvars, but it doesn't need Meson installed, as the calls to meson are being captured by the ``def run()```override.